snackbar/styles2-snackbar.css

```diff
@@ -0,0 +1,53 @@
+/* Snackbar Main */
+.snackbar-container {
+  margin: 3rem;
+  height: 30rem;
+}
+
+.snackbar-main-heading {
+  color: yellow;
+}
+.snackbar-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.snackbar-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
+  height: 200px;
+  border: 0;
+  transform: scale(1);
+  overflow: hidden;
+}
+.standard-snackbar {
+  color: yellow;
```

Comment:
These tend to appear different in different browsers
Use color variables with hex code

Reply:
okayy done